microcosm_pubsub/tests/test_sentry.py

```diff
@@ -0,0 +1,83 @@
+"""
+Sentry integration tests.
+
+"""
+from hamcrest import assert_that, equal_to, is_
+from microcosm.api import create_object_graph
+from sentry_sdk.utils import BadDsn
+
+from microcosm_pubsub.sentry import before_send
+from microcosm_pubsub.tests.sentry_fixture import sample_event
+
+
+def test_sentry_config_bad_dsn():
+    def loader(metadata):
+        return dict(
+            sentry_logging=dict(
+                enabled=True,
+                dsn="invalid schema",
+            ),
+            build_info_convention=dict(
+                sha1="some-sha",
+            )
+        )
+
+    graph = create_object_graph("example", testing=False, loader=loader)
+    try:
+        graph.sentry_logging.enabled
+        assert False
```

Comment:
I think this is a bit more...expressive?
`assert_that(calling(lambda: graph.sentry_logging.enabled), raises(BadDsn))`

Reply:
Oddly enough that wasn't working as it's what I tried first. Updating the test to handle `BadDsn` and disable sentry in this case.